sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/MapsRenderServiceVersion.java

```diff
@@ -31,7 +31,7 @@ public enum MapsRenderServiceVersion implements ServiceVersion {
      * @return the latest {@link MapsRenderServiceVersion}
      */
     public static MapsRenderServiceVersion getLatest() {
-        return V2_1;
+        return V2;
```

Comment:
Why it's changed from `V2_1` to `V2`? Shouldn't it be `V2024_04_01` if we follow the same naming rule?

Reply:
This is how blob storage name their version: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobServiceVersion.java

Reply:
`        internal const ServiceVersion LatestVersion = ServiceVersion.V2_0;

        /// <summary> The version of the service to use. </summary>
        public enum ServiceVersion
        {
            /// <summary> Service version "1.0". </summary>
            V1_0 = 1,
            /// <summary> Service version "2.0". </summary>
            V2_0 = 2,
        }`
        
       here what we did for C#, I followed that

Reply:
@dubiety

Reply:
I think we should consult Java SDK archboard.